backend/tests/Notifo.Domain.Tests/appsettings.json

```diff
@@ -1,47 +1,51 @@
 {
-  
```

Comment:
What are all these changes? I think VS fucks up the formatting sometimes.

Reply:
Btw: now those formatting issues are fixed. You can't see it here in this commit, but if you compare it to the current version from the main branch, the indentation is similar.